### PR TITLE
Add Contributing section to Readme

### PR DESCRIPTION
**Pull request by trybick**
I can add more or less detail to specific steps if needed. Here's a screenshot of what it looks like.

Screen Shot 2019-10-06 at 4 41 56 PM

### CODE REVIEW

**Comment by gokatz** [comment]
I'm away from my desktop. Will review this soon.

**Comment by gokatz** [comment]
Thanks for pitching in @trybick. Yes, somewhat detailed step would be great. Please refer to this PR (#16) for details. 

Also, can we move the contribution guide to separate `.md` file (say, `contributing.md`) and link that from `README.md`?

**Comment by trybick** [comment]
@gokatz Will do exactly that (might not have a chance until later today tho)

**Comment by gokatz** [comment]
Not an issue @trybick.

**Comment by trybick** [comment]
@gokatz Created a separate file. Here is a preview picture.

Screen Shot 2019-10-07 at 7 55 30 PM

File: README.md [hunk 1]
```diff
@@ -11,13 +11,7 @@
...
 ## Contributing
 
-This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
-
-- Fork and Clone the Project
-- Create your Feature Branch (git checkout -b feature/amazing-feature)
-- Commit your Changes (git commit -m 'Add some awesome stuff')
-- Push to the Branch (git push origin feature/amazing-feature)
-- Open a Pull Request
+This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome! Get started [here](https://github.com/gokatz/ua-parser-extension/blob/master/CONTRIBUTING.md)!
```
Comment (gokatz) [inline README.md]: Can we change the 'get started here' text to 'checkout the contribution guide for more details' for high visibility?

**Comment by gokatz** [comment]
Awesome. Looks cool. Just added a small suggestion. Can you look into it ?

**Comment by trybick** [comment]
Will do!

**Comment by trybick** [comment]
@gokatz Updated with your text change. How do you want to handle the merge conflict with the existing `Contribution` section?

**Comment by gokatz** [comment]
Hi @trybick. Thanks for updating the content. This PR #16 also introduced changes to `README.md`. Hence the conflict. Are you familiar with conflicts ?

**Comment by gokatz** [comment]
I've resolved the conflicts and added Prerequisites section in Contribution guide. Can you check if that is okay? Will merge that if you okay with the changes.

**Comment by trybick** [comment]
@gokatz LGTM 🎉  (just wasn't sure how we wanted it structured)

**Comment by gokatz** [comment]
@all-contributors please add @trybick for documentation

**Pull request merged**